### PR TITLE
POKE-2884 add support for quick lookup

### PR DESCRIPTION
**Pull request by nhalle**
Adds support for the new quick lookup endpoint

### CODE REVIEW

File: test/Influencers/InfluencersTest.php [hunk 1]
```diff
@@ -915,4 +915,32 @@ public function testSearch()
             'influencers' => array($this->infUid, $this->infUid2),
             'tags' => array($this->infTag, $this->infTag2)));
     }
+
+    public function testQuickLookup()
```
Comment (tyrantkhan) [inline test/Influencers/InfluencersTest.php:919]: nit: Add a test case to handle when query is not set or empty

**Review by tyrantkhan — APPROVED** [review]
